services/migrations/gitea_downloader.go

```diff
@@ -664,18 +664,26 @@ func (g *GiteaDownloader) GetReviews(reviewable base.Reviewable) ([]*base.Review
 				})
 			}
 
-			allReviews = append(allReviews, &base.Review{
-				ID:           pr.ID,
-				IssueIndex:   reviewable.GetLocalIndex(),
-				ReviewerID:   pr.Reviewer.ID,
-				ReviewerName: pr.Reviewer.UserName,
-				Official:     pr.Official,
-				CommitID:     pr.CommitID,
-				Content:      pr.Body,
-				CreatedAt:    pr.Submitted,
-				State:        string(pr.State),
-				Comments:     reviewComments,
-			})
+			review := &base.Review{
+				ID:         pr.ID,
+				IssueIndex: reviewable.GetLocalIndex(),
+				Official:   pr.Official,
+				CommitID:   pr.CommitID,
+				Content:    pr.Body,
+				CreatedAt:  pr.Submitted,
+				State:      string(pr.State),
+				Comments:   reviewComments,
+			}
+
+			if pr.Reviewer != nil {
+				review.ReviewerID = pr.Reviewer.ID
+				review.ReviewerName = pr.Reviewer.UserName
+			} else {
+				// we have to skip this review as it will be mapped on to something incorrect
+				continue
+			}
+
+			allReviews = append(allReviews, review)
```

Comment:
```suggestion
			// test if we have to skip this review as it will be mapped on to something incorrect
			if pr.Reviewer != nil {
				review.ReviewerID = pr.Reviewer.ID
				review.ReviewerName = pr.Reviewer.UserName
				allReviews = append(allReviews, review)
			}
```

Reply:
The code structure is deliberate.

If we can figure out how to handle this nil case properly then we would want this structure.

```suggestion
			if pr.Reviewer == nil {
				// Presumably this is a team review which we cannot migrate at present but we have to skip this review as otherwise the review will be mapped on to an incorrect user.
				// TODO: handle team reviews
				continue
			}
			review.ReviewerID = pr.Reviewer.ID
			review.ReviewerName = pr.Reviewer.UserName
			allReviews = append(allReviews, review)
```